sig-architecture/api-review-process.md

```diff
@@ -0,0 +1,245 @@
+# Kubernetes API Review Process
+
+[API Approvers](mailto:kubernetes-api-reviewers@googlegroups.com): [Clayton Coleman](https://github.com/smarterclayton), [Jordan Liggitt](https://github.com/liggitt), [Tim Hockin](https://github.com/thockin), [Brian Grant](https://github.com/bgrant0607), [Eric Tune](https://github.com/erictune)
+
+# Process Overview and Motivations
+
+Due to the importance of preserving usability and consistency in Kubernetes APIs, all changes and additions require expert oversight. The API review process is intended to maintain logical and functional integrity of API implementations over time, the consistency of user experience and the ability of previously written tools to function with new APIs. Wherever possible, the API review process should help change submitters follow [established conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md), and not simply reject without cause. 
+
+Because expert reviewer bandwidth is extremely limited, the process provides a curated backlog with highest priority issues at the top. While this does mean some changes may be delayed in favor of other higher priority ones, this will help maintain critical project velocity, transparency, and equilibrium. Ideally, those whose API review priority is shifted in a release-impacting way will be proactively notified by the reviewers.
+
+# Goals of the process
+
+* Provide an easily-navigable process so all parties understand their roles, responsibilities, and expectations
+
+* Protect Kubernetes APIs from disruptive, inconsistent, or destabilizing changes
+
+* Respect, gate, and expand expert reviewer bandwidth while maintaining consistent flow across the process
+
+* Provide transparency, including clear feedback and path forward for API contributors
+
+* Maintain the high standards of the project, including positive user interactions with APIs
+
+* Provide review regardless of method of API defininition (built-in, Extension API Server, or Custom Resource Definition)
+
+* Provide review over both tightly coupled external projects and in-tree API changes.  
+
+* Provide a platform for commenting on popular extensions' APIs, so as to encourage the spread of good API practices throughout the ecosystem
+
+# Non-goals of the process
+
+* Creation or maintenance of the API standards documentation
+
+* Being a "rubber stamp" for changes
+
+# Process Description
+
+![image alt text](image_0.png)
+
+
+## Intake Pre-review Checklist
+
+* The change/addition has been reviewed by the appropriate sub-project stakeholders and SIG chairs as needed, or else the request may be rejected pending initial review
+
+* A KEP has been created if introducing:
+
+    * Any new resource type
+
+    * Any new version of a stable API
+
+    * Any major new functionality added to a stable API as defined by SIG Architecture and the API Reviewers
```

Comment:
Can we change "Any major new" to "Any change in"?  I don't want to get into defining "major", and it is more than just "new"